src/Symfony/Bundle/WebProfilerBundle/Controller/RouterController.php

```diff
@@ -36,11 +37,17 @@ class RouterController
     private $matcher;
     private $routes;
 
-    public function __construct(Profiler $profiler = null, Environment $twig, UrlMatcherInterface $matcher = null, RouteCollection $routes = null)
+    /**
+     * @var ExpressionFunctionProviderInterface[]
+     */
+    private $expressionLanguageProviders = [];
+
+    public function __construct(Profiler $profiler = null, Environment $twig, UrlMatcherInterface $matcher = null, iterable $expressionLanguageProviders = [], RouteCollection $routes = null)
```

Comment:
I can find where/when the $routes argument is set. So i add the new one just befrore

Reply:
please add it last, that's a BC break otherwise

Reply:
How do I define argument "$routes" in profiler.xml ?

Reply:
I declare routes explicitly to null